src/dual_equality_constraints.jl

```diff
@@ -139,4 +139,42 @@ function fill_scalar_affine_terms!(scalar_affine_terms::Vector{MOI.ScalarAffineT
         i += 1
     end
     return 
+end
+
+function fill_scalar_affine_terms!(scalar_affine_terms::Vector{MOI.ScalarAffineTerm{T}},
+                                   primal_con_dual_var::Dict{CI, Vector{VI}},
+                                   primal_model::MOI.ModelLike, ci::CI{F, S}, 
```

Comment:
You can just write `CI{<:VVF, <: MOI.SecondOrderCone}` if you don't use `F` and `S` anywhere